ui/src/components/MultiInputComponent/MultiInputComponent.test.tsx

```diff
@@ -35,6 +47,19 @@ const renderFeature = (additionalProps?: Partial<MultiInputComponentProps>) => {
     render(<MultiInputComponent {...props} />);
 };
 
+const setConfig = () => {
+    const mockConfig = getGlobalConfigMock();
+    setUnifiedConfig(mockConfig);
+};
+
+const mockAPI = () => {
+    server.use(
+        http.get('/demo_addon_for_splunk/some_API_endpint_for_select_data', () =>
```

Comment:
I would extract the string to the file-scoped variable, the literal is used 3 times

Reply:
done